test/integration/component/test_network_permissions.py

```diff
@@ -758,3 +758,24 @@ def test_04_deploy_vm_for_other_user_and_test_vm_operations(self):
         command = """self.reset_network_permission({apiclient}, self.user_network, expected=True)"""
         self.exec_command("self.otheruser_apiclient", command, expected=False)
         self.exec_command("self.user_apiclient", command, expected=True)
+
+    @attr(tags=["advanced"], required_hardware="false")
+    def test_05_list_networks_under_project(self):
```

Comment:
- is this test run against simulator (check travis yml)
- if not, can we move some sort of quick validation tests to smoke/

Reply:
Moved this to smoke tests, I see some tests require hardware.